yucca/preprocessing/YuccaPreprocessor.py

```diff
@@ -363,36 +376,42 @@ def preprocess_case_for_inference(self, images: list | tuple, patch_size: tuple)
         assert isinstance(images, (list, tuple)), "image(s) should be a list or tuple, even if only one " "image is passed"
         self.initialize_properties()
         image_properties = {}
-        images = [nib.load(image[0]) if isinstance(image, tuple) else nib.load(image) for image in images]
+        ext = images[0][0].split(os.extsep, 1)[1] if isinstance(images[0], tuple) else images[0].split(os.extsep, 1)[1]
+        images = [read_any_file(image[0]) if isinstance(image, tuple) else read_any_file(image) for image in images]
 
-        image_properties["original_spacing"] = get_nib_spacing(images[0])
+        image_properties["image_extension"] = ext
         image_properties["original_shape"] = np.array(images[0].shape)
-        image_properties["qform"] = images[0].get_qform()
-        image_properties["sform"] = images[0].get_sform()
 
         assert len(image_properties["original_shape"]) in [
             2,
             3,
         ], "images must be either 2D or 3D for preprocessing"
 
-        # Check if header is valid and then attempt to orient to target orientation.
-        if (
-            images[0].get_qform(coded=True)[1]
-            or images[0].get_sform(coded=True)[1]
-            and self.plans.get("target_coordinate_system")
-        ):
-            image_properties["reoriented"] = True
-            original_orientation = get_nib_orientation(images[0])
-            image_properties["original_orientation"] = original_orientation
-            images = [
-                reorient_nib_image(image, original_orientation, self.plans["target_coordinate_system"]) for image in images
-            ]
-            image_properties["new_orientation"] = get_nib_orientation(images[0])
-        else:
-            print("Insufficient header information. Reorientation will not be attempted.")
-            image_properties["reoriented"] = False
+        image_properties["original_spacing"] = np.array([1.0] * len(image_properties["original_shape"]))
+        image_properties["qform"] = None
+        image_properties["sform"] = None
+        image_properties["reoriented"] = False
+        image_properties["affine"] = None
+
+        if isinstance(images[0], nib.Nifti1Image):
```

Comment:
Images[0] is simply the first image in a list of images. We use that one for the tests because they'll all be the same size and we know we always have minimum one image.

Do you feel like it makes it much easier if we pull it out and call it e.g. image = images[0]? 
for me it doesn't do much

Reply:
I would prefer `first_image` then :-)

Reply:
I think images[0] is perfectly fine, the name makes it clear that it is a list of images 🙈